internal/settings/metrics.go

```diff
@@ -0,0 +1,58 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package settings
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	settings "github.com/onflow/flow-cli/internal/settings/globalSettings"
+)
+
+var MetricsSettings = &cobra.Command{
+	Use:       "metrics",
+	Short:     "Configure command usage metrics settings",
+	Example:   "flow config metrics disable \nflow config metrics enable",
+	Args:      cobra.ExactArgs(1),
+	ValidArgs: []string{"enable", "disable"},
```

Comment:
extract the values as consts so you can also use them bellow in code for checking

Reply:
Maybe `flow config` with flags ( or subcommands ) here can be a better solution. Something like: 

`flow config --metrics enable` or `flow config metrics enable` ( in case of subcommand validargs can be func I guess ) 

Some advantages: 

- it allows discovery, `flow config` will dump all flags, with descriptions. 
- adding new config with a default value will be just like adding a flag ( single line in struct )

Reply:
Don't need flags, it's already a sub command of settings.